src/nidigital/metadata/functions.py

```diff
@@ -2048,7 +2048,7 @@
     'Initiate': {
         'codegen_method': 'private',
         'documentation': {
-            'description': 'TBD'
+            'description': 'Starts bursting the pattern, causing the NI-Digital session\nto be committed. To stop the pattern burst, call abort\nmethod. If keep alive pattern is bursting when abort\nis called, keep alive pattern will not be stopped. To stop\nthe keep alive pattern, call abort_keep_alive method.'
```

Comment:
Make sure this change is made in the internal metadata too. (This applies to all metadata changes).

Reply:
Internal metadata, you meant here: https://p4.natinst.com/browser/perforce/MI/Digital/PBD/SWCode/niDigitalAPIMetadata/master/source/niDigitalAPIMetadata/python/functions/functions.hapigen#line_338?

Reply:
It will need to go into the `nimi_python_documentation` part of the internal metadata, like we have in nidcpowerAPIMetadata.